py3status/modules/google_calendar.py

```diff
@@ -188,9 +187,9 @@ class Py3status:
     force_lowercase = False
     format = '{events}|\?color=event \u2687'
     format_date = '%a %d-%m'
-    format_event = '[\?color=event {summary}][ ({location})][ {format_timer}]'
-    format_event_full = '[\?color=event {summary}]' +\
-        '[\?color=time  ({start_time} - {end_time}, {start_date})]'
+    format_event = '[\?color=event {summary}][\?if=!event_full [\?if=location  ({location})]]' +\
+        '[\?if=!event_full  {format_timer}]' +\
+        '[\?if=event_full  ({start_time} - {end_time}, {start_date})]'
```

Comment:
This looks a tad long. I think this is same.
```
format_event = '[\?color=event {summary}][\?if=event_full ({start_time}' +\
    ' - {end_time}, {start_date})|[({location})] {format_timer}]'
```
This `[\?if=event_full True|False]` is same as `[\?if=event_full True][if=!event_full False]`.

And we should use `is_toggled` as this is more obvious than `event_full`... and to add `Control placeholders` for `is_toggled`. (See cmus or moc). :+1: